js/latoken_v2.js

```diff
@@ -0,0 +1,895 @@
+'use strict';
+
+//  ---------------------------------------------------------------------------
+
+const Exchange = require ('./base/Exchange');
+const { ExchangeError, ArgumentsRequired, InvalidNonce, OrderNotFound, InvalidOrder, DDoSProtection, BadRequest, AuthenticationError } = require ('./base/errors');
+const { ROUND } = require ('./base/functions/number');
+
+//  ---------------------------------------------------------------------------
+
+module.export = class latoken_v2 extends Exchange {
```

Comment:
The naming convention for the classname and the filename is `latoken2`/`latoken2.js` in this case, can you plz rename it? Also, please, remove the autogenerated files (ccxt.js and ccxt.d.ts) from this PR.

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-contribute-code
- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#transpiled-generated-files